SPECS/perl/gendep.macros

```diff
@@ -198,6 +199,7 @@ Requires: perl(CPAN::Meta::Requirements) >= 2.121 \
 Requires: perl(CPAN::Meta::Validator) \
 Requires: perl(Carp) \
 Requires: perl(Exporter) \
+Requires: perl(Parse::CPAN::Meta) >= 1.4400 \
 Requires: perl(Parse::CPAN::Meta) >= 1.4414 \
```

Comment:
I'm worried this might cause build issues with our tooling for other packages. Since the dependency's name is identical, I'd remove the duplicate, older version:

```suggestion
Requires: perl(Parse::CPAN::Meta) >= 1.4414 \
```

Reply:
Sure, I'll clean these up. For some reason, only the duplicate "perl(:VERSION)" entries caused issues with our tooling